llvm/test/CodeGen/SPIRV/hlsl-intrinsics/asuint16.ll

```diff
@@ -0,0 +1,38 @@
+; RUN: llc -O0 -verify-machineinstrs -mtriple=spirv-unknown-unknown %s -o - | FileCheck %s
+; RUN: %if spirv-tools %{ llc -O0 -mtriple=spirv-unknown-unknown %s -o - -filetype=obj | spirv-val %}
+
+; CHECK-DAG: %[[#half:]] = OpTypeFloat 16

```

Comment:
delete this file. the tests you have in the asint16 pr are sufficient to for testing `OpBitcast` works for both scalars and vectors.